com.yktoo.ymuse.yml

```
@@ -0,0 +1,40 @@
+app-id: com.yktoo.ymuse
+default-branch: master
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: ymuse
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

this assuming it actually support wayland

---

It does, yes